cmd/snap-seccomp/main_test.go

```diff
@@ -177,10 +178,15 @@ func (s *snapSeccompSuite) SetUpSuite(c *C) {
 	err = cmd.Run()
 	c.Assert(err, IsNil)
 
+	if release.DistroLike("amzn") {
+		// Amazon Linux 2 is 64bit only and there is no multilib support
+		s.cannotCheckCompatArch = true
```

Comment:
Nitpick: please invert this so that we don't need `!cannotCheck...` code

Reply:
Per @mvo5 suggestion, I'll force push the update.